pkg/kubelet/cm/topologymanager/topology_manager.go

```diff
@@ -79,6 +92,9 @@ type HintProvider interface {
 	// a consensus "best" hint. The hint providers may subsequently query the
 	// topology manager to influence actual resource assignment.
 	GetTopologyHints(pod *v1.Pod, container *v1.Container) map[string][]TopologyHint
+	// GetPodLevelTopologyHints returns a map of resource names to a list of possible
+	// concrete resource allocations per Pod in terms of NUMA locality hints.
+	GetPodLevelTopologyHints(pod *v1.Pod) map[string][]TopologyHint
```

Comment:
nit: I'm curious if the term `Level` or `Scope` should be used in the name of this method.

Reply:
I understand that mixing two terms is not a good idea. I considered if it is necessary to have `Level` or `Scope` in that function. I think it's not so I renamed the function to `GetPodTopologyHints`.